src/Sulu/Component/Content/Document/Subscriber/ResourceSegmentSubscriber.php

```diff
@@ -128,4 +151,16 @@ private function getResourceSegmentProperty($document)
 
         return $property;
     }
+
+    private function persistDocument(ResourceSegmentBehavior $document, PropertyMetadata $property)
```

Comment:
what should the PHPDoc say? I would vote for not PHPDocing `private` methods unless they do something that needs explaining.